tests/test_settings_service.py

```
@@ -221,13 +173,16 @@ class TestSettingsRouterErrors:
     """Router-level tests for settings API error and validation paths."""
 
     @pytest.fixture()
-    async def _db(self, monkeypatch):
+    async def _db(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_db`

(ANN202)